<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StationListView</class>
 <widget class="QMainWindow" name="StationListView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Station List</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListView" name="listView"/>
    </item>
    <item>
     <widget class="QLineEdit" name="filterEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>782</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="sortByNameAction"/>
    <addaction name="sortNearFirstAction"/>
    <addaction name="sortRecentFirstAction"/>
    <addaction name="separator"/>
    <addaction name="showSettingsAction"/>
    <addaction name="showAboutAction"/>
   </widget>
   <addaction name="menuView"/>
  </widget>
  <action name="sortNearFirstAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Near</string>
   </property>
   <property name="toolTip">
    <string>Show near stations first</string>
   </property>
  </action>
  <action name="sortByNameAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ABC</string>
   </property>
   <property name="toolTip">
    <string>Sort stations by name</string>
   </property>
  </action>
  <action name="showSettingsAction">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="showAboutAction">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="sortRecentFirstAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Recent</string>
   </property>
   <property name="toolTip">
    <string>Show recently checked stations first</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
